apps/designer/app/designer/shared/image-manager/image-thumbnail.tsx

```diff
@@ -98,7 +90,8 @@ export const ImageThumbnail = ({
 
   const isUploading = status === "uploading";
 
-  const src = useImageWithFallback({ path });
+  // Must be the same as the Thumbnail width "$spacing$19"
+  const imageWidth = getInt(theme.spacing[19].value) ?? 64;
```

Comment:
hmm, I wonder if there is a better way to reuse the value and keep it in sync

Reply:
same but havent found

Reply:
Put `const THUMBNAIL_SIZE = theme.spacing[19].value` at the top of the file and use it in both places?

Reply:
yep, even if its not a token but it uses token value anyway

Reply:
also you don't need a safe getInt here,since we know its not gonna be something else, so you could just use parseFloat()

Reply:
I was almost suggesting to reuse getInt implementation from the image package, which is also an option, but I would not copy paste getInt

Reply:
just thought about it - define a constant locally and use it in all 3 places, ditch the theme in this case, its not that important, just add a comment that the value should be one of the values in the theme, because of the spacing grid compliance